src/EditorFeatures/CSharp/GoToDefinition/CSharpGoToDefinitionItemService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.GoToDefinition;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.GoToDefinition
+{
+    [ExportLanguageService(typeof(IGoToSymbolService), LanguageNames.CSharp), Shared]
+    internal class CSharpGoToDefinitionItemService : AbstractGoToDefinitionItemService
```

Comment:
i'm going to be annoying.  But you're exporting IGoToSymbolService with something named GotoDefinitionItemService.  These names should in sync.